apps/davi/src/Modules/Guilds/pages/Governance/Governance.styled.tsx

```diff
@@ -13,11 +12,8 @@ export const StyledHeading = styled(Heading)`
   margin-bottom: 20px;
 `;
 
-export const StyledLink = styled(Link)`
-  color: ${({ theme }) => theme.colors.text};
-`;
-
 export const StyledButton = styled(Button)`
   white-space: nowrap;
   height: 45px;
+  border-color: ${({ theme }) => theme.colors.grey3};

```

Comment:
This button styling seems to be repeated various times. Should we extract it into a variant?